config/application.rb

```diff
@@ -43,6 +43,7 @@ class Application < Rails::Application
     # use the cookie session and set the name of the cookie
     config.session_store :cookie_store, key: "SESSION"
 
-    # Application Insights
+    # load our form concerns
+    config.eager_load_paths += %W[#{config.root}/forms/converns]
```

Comment:
Typo `concerns` not `converns`

I'm slightly concerned (lol) that the tests pass/app loads with this typo in it?! Does that mean this line isn't needed?

Reply:
yeah! it probably does!